docs/brandenberg/usecase.md

```diff
@@ -1,89 +1,893 @@
+liquefaction information in a common and comprehensive database to provide all researchers with a 
+substantially larger, more consistent, and more reliable source of liquefaction data than existed previously. 
+Based on this database, we will create probabilistic models that provide hazard- and risk-consistent bases 
+for assessing liquefaction susceptibility, the potential for liquefaction to be triggered in susceptible soils, 
+and the likely consequences. NGL is committed to an open and objective evaluation and integration of data, 
+models and methods, as recommended in a 2016 National Academies [report](https://www.nap.edu/catalog/23474/state-of-the-art-and-practice-in-the-assessment-of-earthquake-induced-soil-liquefaction-and-its-consequences). 
 
-## Background 
+The evaluation and integration of the data into new models and methods will be clear and transparent. Following these principles will ensure 
+that the resulting liquefaction susceptibility, triggering, and consequence models are reliable, robust and 
+vetted by the scientific community, providing a solid foundation for designing, constructing and overseeing 
+critical infrastructure projects.
 
-### Citation and Licensing
+The NGL database is populated through a web GUI at www.nextgenerationliquefaction.org/. The web interface 
+provides limited capabilities for users to interact with data. Users are able to view and download data, 
+but they cannot use the GUI to develop an end-to-end workflow to make scientific inferences and draw conclusions 
+from the data. To facilitate end-to-end workflows, the NGL database is replicated daily to [DesignSafe](https://www.designsafe-ci.org), where 
+users can interact with it using Jupyter notebooks.
 
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
+## Citations and Licensing
+
+* Please cite [Zimmaro, P., et al. (2019)](https://doi.org/10.21222/C2J040) to acknowledge the use of the NGL Database. Data in the NGL database has been gathered from [these](https://nextgenerationliquefaction.org/citations.php) published sources. If you use specific data in the database, please cite the original source.
 
 * Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
 
 * This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).  
 
-### Description 
+## Understanding the Database Schema
+
+The NGL database is organized into tables that are related to each other via keys. To query the database, 
+you will need to understand the organizational structure of the database, called the schema. The database 
+schema is documented at the following URL:
+
+<https://nextgenerationliquefaction.org/schema/index.html>
+  
+## Installing Database Connection Script
```

Comment:
Needs discussion on where this code can be run, i.e., introduce Jupyter

Reply:
I added a section called "Querying Data via Jupyter Notebooks" that provides this context.